en/assembly/vibration_isolation.md

```diff
@@ -0,0 +1,36 @@
+# Vibration Isolation
+
+This topic shows how to determine whether vibration levels are too high, and lists some simple steps to improve vibration performance.
+
+## Overview
+
+Flight Control boards with in-built accelerometers or gyros are sensitive to vibrations.
+High vibration levels can cause sensor clipping and sensor failures, resulting in a range of problems, from reduced flight efficiency/performance (leading to shorter flights and increased vehicle wear-and-tear), through to fly-aways and crashes.
```

Comment:
The way I understand this, the clipping and failures result in a range of problems. However event w/o clipping or failures you can have problems.
Do I understand this incorrect?

Reply:
What this says is that it is the sensor and clipping failures (caused by high vibration) that result in the listed problems.

I think you are saying that some of these problems are not due to the sensor and clipping failures - so you can still have the problems.

I can reword, but would be useful if you can help me understand which problems are associated just with clippin?

Reply:
Exactly. Clipping is the extreme case that leads to (position) estimation failures and fly-aways.
This holds for the accel, I've never seen a gyro clipping, but likely similar bad things happen.

Reply:
Thanks @bkueng . So would you be happy with:

> High vibration levels can cause a range of problems, including reduced flight efficiency/performance, shorter flights and increased vehicle wear-and-tear. In extreme cases vibration may lead to sensor clipping/failures, possibly resulting in estimations and fly-aways.

Reply:
I'd say 'shorter flight time' and 'estimation failures', otherwise looks good.

Reply:
Thanks. Fixed and nearly merged. Those were pretty rubbish typos I hope I would have fixed before submitting.